packages/asset-swapper/contracts/src/SushiSwapSampler.sol

```diff
@@ -0,0 +1,103 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-utils/contracts/src/DeploymentConstants.sol";
+import "./interfaces/IUniswapV2Router01.sol";
+
+
+contract SushiSwapSampler is
+    DeploymentConstants
+{
+    /// @dev Gas limit for SushiSwap calls.
+    uint256 constant private SUSHISWAP_CALL_GAS = 150e3; // 150k
+
+    /// @dev Sample sell quotes from SushiSwap.
+    /// @param router Router to look up tokens and amounts
+    /// @param path Token route. Should be takerToken -> makerToken
+    /// @param takerTokenAmounts Taker token sell amount for each sample.
+    /// @return makerTokenAmounts Maker amounts bought at each taker token
+    ///         amount.
+    function sampleSellsFromSushiSwap(
+        address router,
```

Comment:
encoded the router incase anything changes in the short term. Might remove later idk